evidence/pool.go

```diff
@@ -680,7 +730,15 @@ func prefixToBytes(prefix int64) []byte {
 
 func keyCommitted(evidence types.Evidence) []byte {
 	var height int64 = evidence.Height()
-	key, err := orderedcode.Append(nil, prefixCommitted, height, string(evidence.Hash()))
+	var hash string
+	// if it is light client attack evidence we add the header hash key to avoid
+	// submission of multiple variants
+	if lcae, ok := evidence.(*types.LightClientAttackEvidence); ok {
+		hash = string(lcae.ConflictingBlock.Header.Hash())
+	} else {
+		hash = string(evidence.Hash())
+	}
```

Comment:
```suggestion
	hash := string(evidence.Hash())
	
	// If it is light client attack evidence we add the header hash key to avoid
	// submission of multiple variants.
	if lcae, ok := evidence.(*types.LightClientAttackEvidence); ok {
		hash = string(lcae.ConflictingBlock.Header.Hash())
	}

```